ChangeLog.md

```diff
@@ -17,6 +17,9 @@ Current Trunk
 -------------
  - Add `emscripten_return_address` which implements the functionality of
    gcc/clang's `__builtin_return_address`. (#8617)
+ - Add support for [undefined behavior sanitizer](https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html).
+   This allows `emcc -fsanitize=undefined` to work. (#8651)
+   The minimal runtime (`-fsanitize-minimal-runtime`) also works. (#8617)
```

Comment:
Without sub-bullets, these two lines will be merged into the previous line. You can do like
```
 - Add support for [undefined behavior sanitizer]
   - This allows `emcc -fsanitize=undefined` to work. (#8651)
   - The minimal runtime (`-fsanitize-minimal-runtime`) also works. (#8617)
```